test/integration/targets/ansible-galaxy-collection/library/setup_collections.py

```diff
@@ -258,7 +263,9 @@ def run_module():
         r['build']['rc'] + r['publish']['rc'] for r in result['results']
     ))
 
-    module.exit_json(failed=failed, **result)
+    end = datetime.datetime.now()
+    delta = end - start
+    module.exit_json(failed=failed, end=str(end), delta=str(delta), **result)
```

Comment:
Is keeping the `start`/`end`/`delta` still desired? I know it doesn't really matter but curious if the newer version is slower or temperamental still?